README.md

```diff
@@ -108,12 +108,20 @@ Fine-tuning parameters:
 Please submit your performance results on other GPUs.
 <br >
 
+## 💾 Models
+
+| model           | Path          |
+|-----------------|---------------|
+| GPT-2           | `x/gpt2`      |
+| GPT-2 distilled | `x/gpt2_distill` |
```

Comment:
These are not good names. What dataset were they fine-tuned on? It doesn't say that.